ethan-aws-finops-culture-blog

```diff
@@ -0,0 +1,86 @@
+
+**Building a FinOps-Centric Culture**
+
+Creating a FinOps-centric culture requires a deliberate effort from leadership and a commitment to the following principles:
+
+
+
+* Education and Training: Invest in training and educational resources to ensure that all employees understand the importance of FinOps and their role. Regular workshops and webinars can help build a shared knowledge base.
+* Transparency: Make cloud costs and financial data transparent throughout the organisation. Use dashboards and reports to provide real-time visibility into spending and progress toward budget goals.
+* Incentives: Consider implementing incentive programs that reward teams and individuals for cost-saving ideas and practices. Recognise and celebrate achievements in cost optimisation. There is currently a working group within the foundation focused on gameification FinOps. Recent examples include donating savings to charities. 
+* Leadership Support: Leadership must lead by example. When executives and managers prioritise cost-consciousness and actively participate in FinOps initiatives, it sends a clear message throughout the organisation.
+* Feedback Loop: Establish a feedback mechanism that allows employees to share their insights and suggestions for cost optimisation. Act on this feedback to demonstrate that their input is valued.
+
+**Common Challenges in Embracing the People Side of FinOps**
+
+Despite the importance of people, processes, and culture in FinOps, organisations often face challenges when embracing these aspects fully. Here are some common hurdles and strategies to overcome them:
+
+**Resistance to Change:**
+
+
+
+* Challenge: Employees may resist changes in processes and culture, especially if they perceive them as disruptive.
+* Strategy: Communicate the benefits of FinOps clearly, provide adequate training and support during the transition, and address concerns proactively.
+
+**Lack of Data Visibility**
+
+
+
+* Challenge: Limited visibility into cloud costs can hinder FinOps efforts.
+* Strategy: Implement cost tracking and reporting tools that provide real-time data and make this data easily accessible to relevant teams.
+
+**Short-term Focus and Priorities**
+
+
+
+* Challenge: Organisations may prioritise short-term cost savings over long-term optimisation.
+* Strategy: Educate teams about the benefits of long-term cost optimisation and incentivise practices that lead to sustainable cost management.
+
+**Closing Thoughts: **
+
+While technology undoubtedly plays a significant role in FinOps, it's essential to remember that the true potential of FinOps lies in its human elements: people, processes, and culture. A well-trained FinOps team, efficient processes, and a culture that values cost-consciousness are the driving forces behind successful cost management in the cloud. 
```

Comment:
What is the call to action? what do you want them to do after they read this blog?